flowmachine/flowmachine/core/server/query_schemas/pareto_interactions.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marshmallow import Schema, fields, post_load
+from marshmallow.validate import OneOf, Length
+
+from flowmachine.features import ParetoInteractions
+from .base_exposed_query import BaseExposedQuery
+from .custom_fields import SubscriberSubset
+
+__all__ = ["ParetoInteractionsSchema", "ParetoInteractionsExposed"]
+
+
+class ParetoInteractionsSchema(Schema):
+    query_kind = fields.String(validate=OneOf(["pareto_interactions"]))
+    start = fields.Date(required=True)
+    stop = fields.Date(required=True)
+    proportion = fields.Float(required=True)
```

Comment:
Can we please add a `validate` parameter here to ensure that `proportion` is a value between 0.0 and 1.0?